tox.ini

```diff
@@ -75,6 +78,8 @@ ignore =
     W503,
     # Missing docstring in *
     D10,
+    # Single vs double quotes conflicting with black
+    Q000,
```

Comment:
Can you explain why this is needed? If we enable `flake8-black` or similar, would that help?

I'd prefer that our linting tools *not* conflict with Black, since that's our formatter of choice.

Reply:
We already have `flake8-black` included in our `deps` for `testenv:lint`, and we can't invoke `flake8-black` directly, it works as a plugin to the `flake8` command. Strangely, even with `flake8-black` working, they seem to disagree on single and double quotes _only_ in a `setup.py` file. 

With this rule ignore, though, we are telling flake that we would like to trust black on this one.